### DISCUSSION TITLE
Version 1.28.0

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#570] [Announcements]
- Clock widget can now be configured from home screen, like every other widget
- Added several new variants of the default watch face, including one that uses Material You colors
- Added preference to set clock widget vertical alignment
- Fix adding widgets on Android 14
- Other bug fixes and improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.28.0'>Version 1.28.0</a>.</em>

**Comment by SATISHDEEPU** [comment]
The best application I have used till date. Better than niagra.
